MatrixSDK/MXSession.h

```diff
@@ -398,6 +398,11 @@ FOUNDATION_EXPORT NSString *const kMXSessionNoRoomTag;
  */
 @property (nonatomic, readonly) MXMediaManager *mediaManager;
 
+/**
+ The the maximum size an upload can be in bytes.
+ */
+@property (nonatomic, readonly) NSInteger maxUploadSize;
```

Comment:
I was split between storing this here or in the media manager?

Reply:
MXSession is ok. Media manager should be a seperate component. I would like us to replace the cache it manages by another 3rd party library.